src/jit/morph.cpp

```diff
@@ -1557,6 +1565,14 @@ void fgArgInfo::ArgsComplete()
                 {
                     prevArgTabEntry->needPlace = true;
                 }
+#if defined(_TARGET_ARM_) && !defined(LEGACY_BACKEND)
+                else if (prevArgTabEntry->isSplit)
+                {
+                    // TODO: We need to test it on LegacyJIT/ARM32.
```

Comment:
I'll remove this comment because I did not found any problem on LegacyJIT/ARM32.